cygnus-common/src/test/java/com/telefonica/iot/cygnus/backends/elasticsearch/ElasticsearchBackendImplTest.java

```diff
@@ -149,17 +157,18 @@ public void testBulkInsert_woData() {
         public void testBulkInsert_oneData() {
             System.out.println(getTestTraceHead(String.format("[ElasticsearchBackendImplTest.bulkInsert OK, TLS=%b, Data=1]", flag)));
 
+            System.out.println(String.format("%s, %b, %s", schema, flag, msg));
             List<Map<String, String>> data = new ArrayList<Map<String, String>>();
             data.add(new HashMap<String, String>(){
                 {
                     put("recvTimeTs", "test_recvTimeTs_1");
-                    put("data", "{\"message\": \"test data 1\"}");
+                    put("data", msg);
                 }
             });
-            String recvTimeTs1Hash = "599F4FB6AFDB1BEF51126B47DAC2696F";
+            // String recvTimeTs1Hash = "599F4FB6AFDB1BEF51126B47DAC2696F";
```

Comment:
Leftover comment? In that case, it should be removed.

Reply:
I'm sorry, this is my mistake. I'll remove this line and re-send pull request soon.

Reply:
> ... and re-send pull request soon.

Just to clarify, you could use the same PR, no need of re-sending a new PR. Just commit the fix to the PR branch (and put a `"Fixed in <commithash>"` mark below this comment to resolve it). Thanks!

Reply:
Fixed in 7a6730e